docs/guides/main.rst

```diff
@@ -245,16 +248,33 @@ put the field before the term, separated by a : character. So ``album:bird``
 only looks for ``bird`` in the "album" field of your songs. (Need to know more?
 :doc:`/reference/query/` will answer all your questions.)
 
-The ``beet list`` command has another useful option worth mentioning, ``-a``,
-which searches for albums instead of songs::
+The ``beet list`` command has couple of other useful options worth mentioning.
+
+``-a``, which searches for albums instead of songs::
 
     $ beet ls -a forever
     Bon Iver - For Emma, Forever Ago
     Freezepop - Freezepop Forever
 
+``-f``, which lets you specify what gets displayed in the results of a search::
+
+    $ beet ls -a forever -f "[$format] $album ($year) - $artist - $title"
+    [MP3] For Emma, Forever Ago (2009) - Bon Iver - Flume
+    [AAC] Freezepop Forever (2011) - Freezepop - Harebrained Scheme
+     
+    As you can see, Beets has replaced the fields specified by the ``$`` prefix 
+    (e.g. $format, $year) with values for each item in the results. A full 
+    list of fields available for use can be found by running ``beet fields``. If
+    you'd like beet to use this format as default (without having to use ``-f``)
+    simply add it to your config file like this::
+    
+       format_item: "[$format] $album ($year) - $artist - $title"
+    
+    And yes, the enclosing double-quotes are necessary.
```

Comment:
I like the idea of describing `-f` here, which is very powerful! But I'm concerned that the `format_item` config option here is getting a little too detailed. The "getting started" guide is meant to be very high-level, and we need to avoid describing too much of beets's (vast) set of configuration options.

Is there any chance you could try simplifying this down to just describe the `-f` option?

Also, it looks like these paragraphs need to be "out-dented" to avoid being parsed as code by ReST.